auth-starter/src/main/java/com/vaadin/auth/starter/VaadinAuthContext.java

```diff
@@ -0,0 +1,21 @@
+package com.vaadin.auth.starter;
+
+import org.springframework.security.oauth2.core.oidc.user.OidcUser;
+
+import java.util.Optional;
+
+/**
+ * An interface to access the authentication context of the application
+ *
+ * @author Vaadin Ltd
+ * @since 1.0
+ */
```

Comment:
We can expect this interface to become the main API entry-point for the all the features we provide, so better be more descriptive.

> An interface to access the authentication context of the application. An instance of this interface is available for injection as bean in view and layout classes.